app/views/campaign_requests/_request_details.html.erb

```diff
@@ -2,26 +2,148 @@
+    </p>
+
+    <%= r.input :government_theme,
+                as: :text,
+                label: "Which of the current government priority themes does this campaign website support and how?",
+                required: true,
+                input_html: {:class => "input-md-6",
+                             :rows => 6,
+                             :cols => 50,
+                             :"aria-required" => true } %>
 
-    <%= r.input :start_date, label: "Start date", required: true, input_html: { class: "input-md-2", :"data-module" => 'calendar', :"data-format" => 'dd-mm-yy', :"data-min-date" => 0, :placeholder => "dd-mm-yyyy", :value => r.object.start_date} %>
-    <%= r.input :end_date, label: "Campaign end date / review date (within 6 months of launch)", required: true, input_html: { class: "input-md-2", :"data-module" => 'calendar', :"data-format" => 'dd-mm-yy', :"data-min-date" => 0, :placeholder => "dd-mm-yyyy", :value => r.object.end_date} %>
+    <%= r.input :description,
+                as: :text,
+                label: "Campaign description",
+                required: true,
+                input_html: {:class => "input-md-6",
+                             :rows => 6,
+                             :cols => 50,
+                             :"aria-required" => true } %>
+    <p class="help-block">
+      Please submit your OASIS plan using this template (OASIS template from the GCS website). In particular you need to demonstrate how the digital / website element of the campaign integrates into your overall OASIS plan and we will need to see specific digital KPIs in the evaluation section
+    </p>
 
-    <%= r.input :description, as: :text, label: "Campaign description", required: true, input_html: {:class => "input-md-6", :rows => 6, :cols => 50, :"aria-required" => true } %>
+    <%= r.input :call_to_action,
+                as: :text,
+                label: "Call to action",
+                required: true,
+                input_html: {:class => "input-md-6",
+                             :rows => 6,
+                             :cols => 50,
+                             :"aria-required" => true } %>
 
-    <%= r.input :call_to_action, as: :text, label: "Call to action", required: true, input_html: {:class => "input-md-6", :rows => 6, :cols => 50, :"aria-required" => true } %>
-    <%= r.input :success_measure, as: :text, label: "How will you measure success?", required: true, input_html: {:class => "input-md-6", :rows => 6, :cols => 50, :"aria-required" => true } %>
+    <%= r.input :proposed_url,
+                label: "Proposed URL (in the form of xxxxx.campaign.gov.uk)",
+                required: true, input_html: { class: "input-md-6",
+                                              :"aria-describedby" => "urlHelpBlock",
+                                              :"aria-required" => true } %>
+    <p class="help-block">
```

Comment:
Do we need a `proposed_url_reasons` field for when they've asked for something that's not `xxxx.campaign.gov.uk`, or `xxxx.gov.uk` ?  The help text suggests stating their reasons, but they can't put those reasons in the `proposed_url` field as then it won't validate as a url.

Reply:
Changed to make the `campaign.` part optional: 
`/((http|https):\/\/)?[a-z0-9]+([\-\.]{1}[a-z0-9]+)*\.(campaign\.)?gov.uk?/`

Reply:
Will ask.

Reply:
Have asked. We've removed the part of the help text that suggests you have any other option besides `xxxxx.campaign.gov.uk` or `xxxxx.gov.uk`.